packages/xrpc-server/src/server.ts

```diff
@@ -386,9 +386,11 @@ export class Server {
     const authVerifier = this.createAuthVerifier(cfg)
 
     const { handler } = cfg
+    const { ws } = this.options
     this.subscriptions.set(
       nsid,
       new XrpcStreamServer({
```

Comment:
would it be worth changing the typing of the argument here? It's currently:
```ts
  constructor(opts: ServerOptions & { handler: Handler }) {
```

https://github.com/bluesky-social/atproto/blob/b5afb723be392d236799bbcb6a55956bd12316ba/packages/xrpc-server/src/stream/server.ts#L9